QUARKUS-4430.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-3535 - Add a metric for rejected submissions to the pool in RHBQ 3.2
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4430
+
+PR: https://github.com/quarkusio/quarkus/pull/39583
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/39581
+
+Verify that URIs in metrics tags are correctly recorded when they match a client or server pattern,
+especially in cases of redirection (302) and not found (404) responses.
+
+## Scope of the testing
+* Run Quarkus micrometer Prometheus application with OpenId Connect Keycloak and verify in case of redirections (302)
+  and not found (404) response the Uris metrics tags are correctly recorded.
+* The test coverage will be verified for JVM.
```

Comment:
judging by code changes this will work in native, but if you integrate this into existing test class, you will run it in native anyway (as you should)